src/apps/chifra/cmd/abis/handle_find.go

```diff
@@ -13,12 +13,106 @@
 package abis
 
 import (
+	"bufio"
+	"bytes"
+	"encoding/hex"
 	"fmt"
+	"log"
+	"os"
+	"runtime"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	ants "github.com/panjf2000/ants/v2"
+
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/config"
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/output"
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/utils"
 )
 
 // HandleFind loads manifest, sorts pins and prints them out
-func HandleFind(terms []string) {
-	for i, term := range terms {
-		fmt.Println("term[", i, "]: ", term)
+func HandleFind(args []string) {
+	testMode := utils.IsTestMode()
+	apiMode := utils.IsApiMode()
+	if apiMode || output.Format == "" || output.Format == "none" {
+		output.Format = "json"
+	}
+
+	var arguments [][]byte
+	for _, arg := range args {
+		if arg[:2] == "0x" {
+			asBytes, err := hex.DecodeString(arg[2:])
+			if err != nil {
+				fmt.Printf("Couldn't parse argument %s\n", arg)
+			} else {
+				arguments = append(arguments, asBytes)
+			}
+		}
+	}
+
+	var wg sync.WaitGroup
+	checkOne, _ := ants.NewPoolWithFunc(runtime.NumCPU()*2/3, func(input interface{}) {
```

Comment:
Pool size is a configuration value now. You can get it by calling `config.ReadBlockScrape().Dev.MaxPoolSize`

Reply:
Done